src/translations/main_es.ts

```diff
@@ -0,0 +1,810 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="es_ES">
+<context>
+    <name>AutoUpdater</name>
+    <message>
+        <source>No ssl configuration!
+OpenSSL might not be installed.</source>
+        <translation>¡SSL no configurado!
+OpenSLL podría no estar instalado.</translation>
+    </message>
+    <message>
+        <source>Failed to save the Birdtray installer:
+</source>
+        <translation>Fallo al salvar el instalador de Birdtray:</translation>
+    </message>
+    <message>
+        <source>Failed to download the Birdtray installer:
+</source>
+        <translation>Fallo al descargar el instaladro de Birdtray:</translation>
+    </message>
+    <message>
+        <source>Installer download failed</source>
+        <translation>La descarga del instaldor ha fallado</translation>
+    </message>
+    <message>
+        <source>Invalid redirect: </source>
+        <translation>Redireccionamiento inválido:</translation>
+    </message>
+    <message>
+        <source>Update failed</source>
+        <translation>La actualización ha fallado</translation>
+    </message>
+    <message>
+        <source>Failed to start the Birdtray installer.</source>
+        <translation>Fallo al iniciar el instaldor de Birdtray</translation>
+    </message>
+</context>
+<context>
+    <name>BirdtrayApp</name>
+    <message>
+        <source>A free system tray notification for new mail for Thunderbird</source>
+        <translation>Un sistema libre de notificación para la bandeja el sistema para nuevos correos de Thunderbird </translation>
+    </message>
+    <message>
+        <source>Display the contents of the given mork database.</source>
+        <translation>Mostrar el contenido de la base de datos de mork</translation>
+    </message>
+    <message>
+        <source>databaseFile</source>
+        <translation>databaseFile</translation>
```

Comment:
Well, this will be displayed when you start Birdtray in a terminal with the `--help` parameter:
![Birdtray help output](https://user-images.githubusercontent.com/6966049/80412154-b08e2400-88cd-11ea-8227-cfcf92190f2e.PNG)


I just thought since you translated the other ones (like [`string`](https://github.com/gyunaev/birdtray/pull/339/files/552adf7aeac2f01d8a206bc414f6d061b419b8d5#diff-0368a168aaf3c2e4f92961523c1af38cR58) and [`FILE`](https://github.com/gyunaev/birdtray/pull/339/files/552adf7aeac2f01d8a206bc414f6d061b419b8d5#diff-0368a168aaf3c2e4f92961523c1af38cR86)) you probably want to translate this one too. If you want to translate it, it should be a single word though (or two words without a space), so `Fichero de la base de datos` wouldn't work.